stratum/hal/bin/barefoot/build-bf-sde.sh

```diff
@@ -172,6 +175,31 @@ popd
 
 echo "BF SDE build complete."
 
+echo "Build and install target-syslibs and target-utils."
+TARGET_SYSLIBS_TMP=$(mktemp -d)
+git clone https://github.com/p4lang/target-syslibs.git "$TARGET_SYSLIBS_TMP"
+pushd "$TARGET_SYSLIBS_TMP"
+git checkout "$TARGET_SYSLIBS_COMMIT"
+git submodule update --init
+mkdir -p build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX="$SDE_INSTALL" ..
+make install -j "$JOBS"
+popd
+rm -rf "$TARGET_SYSLIBS_TMP"
+
+TARGET_UTILS_TMP=$(mktemp -d)
+git clone https://github.com/p4lang/target-utils.git "$TARGET_UTILS_TMP"
+pushd "$TARGET_UTILS_TMP"
+git checkout ${TARGET_UTILS_COMMIT}
+git submodule update --init --recursive
+mkdir -p build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX="$SDE_INSTALL" -DSTANDALONE=1 ..
+make install -j "$JOBS"
+popd
+rm -rf "$TARGET_UTILS_TMP"
```

Comment:
I think we have to guard this step behind an if SDE 9.9.0 to prevent linker errors:
```
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'tok_end'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'el_deletestr'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'el_source'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'vis'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'el_getc'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'tok_init'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'el_beep'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'el_push'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'tok_str'
bazel-out/k8-fastbuild/bin/_solib_k8/_U@local_Ubarefoot_Ubin_S_S_Cbfsde___Ubarefoot-bin_Slib/libdriver.so: error: undefined reference to 'el_parse'
```

Reply:
I can confirm that this works for 9.7.2 and 9.9.0